src/gearbox/services/criterion.py

```diff
@@ -15,7 +18,7 @@ async def get_criteria(session: Session) -> CriterionSearchResults:
     return aes
 
 
-async def create_new_criterion(session: Session, input_criterion_info: CriterionCreateIn):
+async def create_new_criterion(session: Session, input_criterion_info: CriterionCreateIn, user_id: int):
```

Comment:
what is this user_id for in the creation of a new criteria?

Reply:
It's for updating the "last_updated_by" field in the staging table

Reply:
If we track that on the create should we track the same on the update on line 87?
`update_criterion(session: Session, criterion: CriterionCreateIn, criterion_id: int) -> CriterionSchema:`

Reply:
Is this user_id the id of the admin user making the changes? If so can we get the id from the authentication instead of passing as a param?

Reply:
Fixed. As per line 87, we are not currently tracking last_updated_by for the criterion table, only for the criterion_staging (this is for tracking adjudication actions). I can add that to the model for the criterion table as well if you'd like. Mabye in another JIRA issue?